basics_of_sst-core/tutorial/Example01.md

```diff
@@ -0,0 +1,436 @@
+        }
+    }
+
+    // Send an event over the link.
+    //
+    ExampleEvent* ev = new ExampleEvent();
+    logger_.verbose(CALL_INFO, INFO, 0x00, "Sending event over the link.\n");
+    link_->send(ev);
+
+    logger_.verbose(CALL_INFO, TRACE, 0x00, "Leaving clock for component id %lu\n", componentId_);
+    return done;
+}
+```
+
+The updated clock event handler consists of two parts.  First the link is polled to see if there are any incoming messages.  If so, the `clockTickCount` variable is incremented and, when the desired number of messages are received, `primaryComponentOKToSendSim()` is called to notify SST that, as far as this component is concerned, it's ok to end the simulation.
+
+After any incoming messages are processed, a message is sent from this component over the link to the connected component, which will process incoming messages in the same manner.  Eventually both components will notify SST it's ok to end the simulation and the simulation will end.
+
+Finally, add a reference to the `ExampleEvent.h` include file at the top of the file.
+
+```
+#include "ExampleComponent.h"
+#include "ExampleEvent.h"
+#include <iostream>
+```
+
+## Configuring the Simulation
+
+The updated simulation configuration file is listed below.
+
+```
+# Execute from the command line with the command:
+#   sst exampleConfig.py 2>&1 | tee test.log
+#
+import sst
+
+# Initialize local variables.
+#
+clockTicks = "5"
+clock = "1GHz"
+debug = "2"     #0 = FATAL, 1 = WARN, 2 = INFO, 3 = DEBUG, 4 = TRACE, 5 = ALL
+
+componentName0 = "example_00"
+componentName1 = "example_01"
+
+# Define the component.
+#
+# The parameters are a dictionary and can be any key/value pair defined
+# by the component itself.
+#
+# The second parameter is <library>.<registered_name>
+#
+obj0 = sst.Component(componentName0, "example.ExampleComponent")
+obj0.addParams({
+    "clock"      : clock,
+    "clockTicks" : clockTicks,
+    "debug"      : debug
+    })
+
+obj1 = sst.Component(componentName1, "example.ExampleComponent")
+obj1.addParams({
+    "clock"      : clock,
+    "clockTicks" : clockTicks,
+    "debug"      : debug
+})
+
+# Connect the objects to each other.
+#
+link = sst.Link("link0")
+link.connect((obj0, "link_", "5ns"), (obj1, "link_",  "5ns"))
+
+```
+
+For the most part, it's the same as that from `Example00`, except for the addition of a second component and establishment of a link between the two.
+
+The link is established using the link `connect` method.  The `connect` method takes two parameters, each of them a tuple containing values used to configure the link coming from a component.  
+
+The first value in the tuple is the component connecting to the link.  In this case, a link is being established between components obj0 and obj1.
+
+The second value in the tuple is the identifier used to refer to the link within the component.  This is the same value as that defined in the `SST_ELI_DOCUMENT_PORTS` macro in the component include file.
```

Comment:
"The second value in the tuple is the port within the component to which to connect this link. This is the name given to the port in the ..."